whelktool/scripts/cleanups/2022/02/lxl-3785-supplementTo-isIssueOf.groovy

```diff
@@ -1,60 +1,64 @@
-/* 
-In order to only check new records specify file where timestamp of last run is saved with:
+/***************************************************************************************
+
+SYSTEM PARAMETERS:
+
 -Dlast-run-file=</path/to/file>
+    If given, a timestamp of this run is saved in this file, and used to only
+    select new records on the next run.
 
-(Schedule this script to run continuously until Mimer MODS conversion is fixed)
+****************************************************************************************
 
-***********************************************************************************************************************
+PURPOSE: Clean up newspaper (dagstidningar + tidskrifter) shapes.
 
-Clean up newspaper (dagstidningar + tidskrifter) shapes.
-Link digitized newspaper monographs (issues) to their series. That is, replace supplementTo and/or isPartOf with isIssueOf.
+(Schedule this script to run continuously until Mimer MODS conversion is fixed.)
 
-Don't touch "Projects" and "Channel records" in supplementTo for now.
+ACTIONS:
 
-Example
-Before
-...
-bf2:title [
-    a bf2:Title ;
-    bf2:mainTitle "DAGENS NYHETER  1900-05-28"
-    ] ;
-...
-bf2:supplementTo [
-    a bf2:Instance ;
-    bf2:instanceOf [
-      a bf2:Work ;
-      bf2:contribution [
-        a bflc:PrimaryContribution ;
-        bf2:agent [
-          a bf2:Agent ;
-          sdo:name "DAGENS NYHETER"
-          ]
-        ]
-      ] ;
-    :describedBy [
-      a :Record ;
-      :controlNumber "13991099"
-      ] ;
-    bf2:identifiedBy [
-      a bf2:Issn ;
-      rdf:value "1101-2447"
-      ] , [
-      a bf2:Strn ;
-      rdf:value "http://libris.kb.se/resource/bib/13991099"
-      ]
-    ] ;
-...
-
-After
-...
-bf2:title [
-    a bf2:Title ;
-    bf2:mainTitle "DAGENS NYHETER  1900-05-28"
-    ] ;
-...
-kbv:isIssueOf <https://libris.kb.se/m5z2w4lz3m2zxpk#it> ;
-...
+This performs one or both of these changes (depending on what is applicable per record):
 
+1. Link digitized newspaper monographs (issues) to their series. That is,
+   replace :supplementTo and/or :isPartOf with :isIssueOf.
+
+2. Change paque signe codes in :editionStatement to links newspaper editions
```

Comment:
```suggestion
2. Change opaque signe codes in :editionStatement to links newspaper editions
```

Reply:
Good call; amended and force-pushed (just rebased on develop).